fetch-api/fetch-referrer-policy.js

```diff
@@ -0,0 +1,9 @@
+var baseUrl = 'users.json';
+var policies = ['no-referrer', 'no-referrer-when-downgrade', 'origin',
+  'origin-when-cross-origin', 'unsafe-url'];
+
+policies.forEach(policy => {
+  var url = `${baseUrl}?policy=${policy}`;
```

Comment:
Agreed that this could be confusing. Since my guidance is to "look at DevTools" to see what the request looks like, I thought that including the policy in the URL would help identify each specific request.

(It would be easier if we had some way of getting the underlying `Request` object that resulted in a `Response` via `fetch()`, but I don't think that's possible.)

I'll remove the URL parameter.